### DISCUSSION TITLE
RFE for ci

### DISCUSSION THREAD

**Discussion by Man2Dev** [curl/curl#13595] [Ideas]
add https://github.com/boyter/scc ci as to get interesting statistics

**Comment by dfandrich** [comment]
There are lots of statistics about curl at https://curl.se/dashboard.html but only a few regarding the source code directly.

**Comment by bagder** [comment]
That seems to be a tool for counting lines of code. I don't think we need to run that as a CI job and we already provide graphs for lines of code in the dashboard as @dfandrich mentions above.
